javav2/example_code/iam/src/main/java/com/example/iam/AttachRolePolicy.java

```diff
@@ -1,4 +1,4 @@
-//snippet-sourcedescription:[AttachRolePolicy.java demonstrates how to attach a policy to an existing IAM role.]
+//snippet-sourcedescription:[AttachRolePolicy.java demonstrates how to attach a policy to an existing AWS Identity and Access Management (AWS IAM) role.]

```

Comment:
AWS Identity and Access Management (IAM)
(short name has no 'AWS')

Reply:
Fixed

Reply:
Copyright added to test files too